todo-frontend-angular/src/app/todolist-selector/todo.service.ts

```diff
@@ -13,11 +12,21 @@ export class TodoService {
   private readonly todolistUrl = 'https://localhost:7058/TodoList'
   private readonly todoUrl = 'https://localhost:7058/Todo'
 
+  private readonly todos = new BehaviorSubject<Todo[]>([])
+
+  public get todos$() {
+    return this.todos.asObservable();
+  }
+
   constructor(private http: HttpClient) {
   }
 
   getTodos(id: string) {
-    return this.http.get<Todolist>(`${this.todolistUrl}/${id}`).pipe(map(todolist => todolist.todos));
+    return this.http.get<Todolist>(`${this.todolistUrl}/${id}`)
+      .pipe(
+        map(todolist => todolist.todos),
+        tap((todos: Todo[]) => this.todos.next(todos))
```

Comment:
I dont think you actually need to manually type todos here :) IT will have the type that todoList.todos have